x-pack/legacy/plugins/monitoring/deprecations.js

```diff
@@ -63,5 +63,9 @@ export const deprecations = () => {
         }
       }
     },
+    renameFromRoot(
```

Comment:
shouldn't it be `rename('xpack_api_polling_frequency_millis', 'licensing.api_polling_frequency')`?

Reply:
Yes, I can use `rename` for sure